### PR TITLE
SRVKP-4361: enable results from sidecar logs for pipelines in prod - p01 only

### PR DESCRIPTION
**Pull request by gabemontero**
see https://redhat-internal.slack.com/archives/C02CTEB3MMF/p1717120576571899 

@scoheb @lcarva @stuartwdouglas 

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: components/pipeline-service/production/base/update-tekton-config-features.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  path: /spec/pipeline/results-from
+  # default upstream setting
+  # value: termination-message
+  value: sidecar-logs
+
+- op: add
+  path: /spec/pipeline/max-result-size
+  # default upstream setting
+  # value: 4096
+  value: 12288
```
Comment (gabemontero) [inline components/pipeline-service/production/base/update-tekton-config-features.yaml:12]: @lcarva is looking at the bootc pipelinerun logs to see if this size is sufficient

Comment (lcarva) [reply components/pipeline-service/production/base/update-tekton-config-features.yaml:12]: Comparing what is in taskResults vs the container termination message, it looks like the result almost made it. Only the last image ref was cut off. The new size should be enough. Looking at the SBOM would be an extra guarantee that there aren't any other entries that are just not showing up, but I don't have access to it.

I think we should go ahead with this change. It won't make the size smaller. We may have to bump the size again, but that seems like a reasonable thing to do.

**Comment by gabemontero** [comment]
/hold

@lcarva had an idea about tweaking the bootc dockerfile to work around the result size issue that @scoheb is going to try first before we merge this

**Comment by scoheb** [comment]
Containerfile update to use the same registries throughout has fixed the results too large issue.

**Comment by gabemontero** [comment]
per discussion on the June 4 Konflux community calls we are going to hit the pause button on enabling sidecar in prod, most likely in favor of https://issues.redhat.com/browse/STONEBLD-2326 @lcarva @ralphbean 

I'll revert the setting in stage soon

@jkhelil FYI

**Pull request closed without merging**